mne/decoding/csp.py

```diff
@@ -90,21 +96,24 @@ class CSP(TransformerMixin, BaseEstimator):
            Klaus-Robert Müller. Optimizing Spatial Filters for Robust EEG
            Single-Trial Analysis. IEEE Signal Processing Magazine 25(1), 41-56,
            2008.
-    .. [3] Grosse-Wentrup, Moritz, and Martin Buss. Multiclass common spatial
+    .. [3] Moritz Grosse-Wentrup, Martin Buss. Multiclass common spatial
            patterns and information theoretic feature extraction. IEEE
-           Transactions on Biomedical Engineering, Vol 55, no. 8, 2008.
+           Transactions on Biomedical Engineering 55(8), 2008.
+    .. [4] Alexandre Barachant, Stephane Bonnet, Marco Congedo, Christian
+           Jutten. Common Spatial Pattern revisited by Riemannian Geometry.
+           IEEE International Workshop on Multimedia Signal Processing (MMSP),
+           p. 472, 2010.
```

Comment:
It would be nice to transition this to the `.. footbibliography` format, feel free to do so here if you want or we can tackle it later.

Reply:
Done